### ISSUE TITLE
Improve how the design system team documents decisions

### ISSUE THREAD

**Issue by HelenOsg** [dwp/design-system#169]
## What

Improve how we document decisions made about the design system work.

## Why
Decisions are made in a number of ways and in different places but there is no single place they are held or updated. This makes it hard to follow decision-making process, understand requirements and ends up with work being delayed or incomplete.

## Done when

- [x] Decide location for decisions

**Comment by martinwake** [comment]
Would the Wiki be a suitable place for this? https://github.com/dwp/design-system/wiki

**Comment by jonhurrell** [comment]
We are capturing the shaping and forming of things in confluence.
